controllers/suite_test.go

```diff
@@ -205,14 +206,23 @@ func TestMain(m *testing.M) {
 		panic(fmt.Sprintf("Failed to create a test registry server: %v", err))
 	}
 
-	managed.InitManagedTransport(logr.Discard())
+	fg := feathelper.FeatureGates{}
+	fg.SupportedFeatures(map[string]bool{
+		features.GitManagedTransport: true,
+	})
+
+	err = managed.InitManagedTransport(logr.Discard())
+	if err != nil {
+		panic(fmt.Sprintf("Failed to register managed transports; %v", err))
```

Comment:
```suggestion
		panic(fmt.Sprintf("failed to register managed transports: %v", err))
```

Reply:
This is in alignment with the rest of the panic errors in this file.
I'd prefer to move all such code into a separate function so that we don't have to write so many panic calls, similar to what's done for setting up OCI registry. But maybe separately.